### DISCUSSION TITLE
Stop splitting based on splits statistics.

### DISCUSSION THREAD

**Discussion by altrabio** [kogalur/randomForestSRC#383] [Q&A]
Thanks for this wonderful package.

I would like to have a tree growing procedure that would stop to split a node if all split statistics are below a threshold (for instance if information gain is below 0). Is there anyway to do that ? (Am I missing something?)
I would even be interested in a method implying to define a custom split function.
Thanks for your feedback

**Comment by altrabio** [comment]
Note that a solution would be to modify the C code (randomForestSRC.c file)  but I was wondering if that can be made in another way.
Thanks

**Comment by kogalur** [comment]
Unfortunately there isn't a way to control growth using the split statistic.  We are working on that idea in an upcoming sister package, but we are still many months away from production. 

As for your second question regarding custom split rules, I would direct you to 

https://github.com/kogalur/randomForestSRC/discussions/197#discussion-3851083

In particular, take a look at splitCustom.c in the package.  We hope to have a vignette that will be make the functionality a little easier to use.  But in the mean time, feel free to continue here with any additional questions regarding custom splitting.

**Reply by pej** [reply]
Thanks for the feedback. I finally ended up with using customization with splitCustomt.c and some slight modification into random Forest.c.
